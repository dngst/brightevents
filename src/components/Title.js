import styled from "styled-components";

const Title = styled.h1`
  text-transform: capitalize;
`;
export default Title;
