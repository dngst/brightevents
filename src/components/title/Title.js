import styled from "styled-components";

const Title = styled.h1`
  text-transform: capitalize;
  font-size: ${props => props.fsize };
  color: ${props => props.color };
  padding: ${props => props.padding };
  margin: ${props => props.margin };
` 
export default Title;
