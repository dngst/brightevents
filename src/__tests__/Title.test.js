import { render } from "@testing-library/react";
import Title from "components/Title";

it("works", () => {
  const { container } = render(<Title />);
  expect(container.firstChild).toMatchSnapshot();
});

it("applies default styles", () => {
  const { container } = render(<Title />);
  expect(container.firstChild).toHaveStyleRule("text-transform", "capitalize");
  expect(container.firstChild).toHaveStyleRule("font-size", undefined);
});
