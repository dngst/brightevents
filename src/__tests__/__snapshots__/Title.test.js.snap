// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
.c0 {
  text-transform: capitalize;
}

<h1
  class="c0"
/>
`;
